app/lib/cfe_connection.rb

```diff
@@ -0,0 +1,61 @@
+class CfeConnection
+  CFE_HOST = "https://check-financial-eligibility-staging.cloud-platform.service.justice.gov.uk/".freeze
+
+  class << self
+    def connection
+      CfeConnection.new
+    end
+  end
+
+  def create_assessment_id
+    create_request = {
+      client_reference_id: "EFE-REFERENCE-ID",
```

Comment:
should this ID increment with each call? or is this more like a placeholder until we decide on what we send to CFE?

Reply:
we don't have a client reference - and it usage is unclear (e.g. the API doesn't seem to allow retrieval of old applications by client reference) and I think its mandatory